src/test/java/seedu/todo/testutil/CollectionUtil.java

```diff
@@ -0,0 +1,26 @@
+package seedu.todo.testutil;
+
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.Set;
+
+//@@author reused
+
+/**
+ * Utility methods related to Collections
+ */
+public class CollectionUtil {
+    /**
+     * Returns true if every element in a collection are unique by {@link Object#equals(Object)}.
+     */
+    public static boolean elementsAreUnique(Collection<?> items) {
+        final Set<Object> testSet = new HashSet<>();
```

Comment:
There's an easier way. Just convert the collection to a set and check if the length are the same! 😆